src/BottomNavigation/BottomNavigationAction.react.js

```diff
@@ -42,6 +42,7 @@ const propTypes = {
     disabled: Text.propTypes.style, // eslint-disable-line
   }),
   disabled: PropTypes.bool,
+  hasUpdatedLoadingStates: PropTypes.bool,
```

Comment:
We will be passing this in from `material-components-library` [here](https://github.com/AdaloHQ/material-components-library/blob/main/src/TabNavigator/index.js#L69). I haven't opened that PR yet because I'll need install a new version of this package once this is merged in.

Reply:
We will be passing this in from `material-components-library` [here](https://github.com/AdaloHQ/material-components-library/blob/main/src/TabNavigator/index.js#L69). I haven't opened that PR yet because I'll need install a new version of this package once this is merged in.

Reply:
Actually, I'm going to change the prop name. I don't think library should be aware of feature flags - it should just take a prop that determines whether it shows a loading state or not.